File: Algoritmos e Logica de Programacao/C/Estrutura Sequencial/ExerciciosEstruturaSequencial/ProblemaIdades.c
/*Problema "idades"Fazer um programa para ler o nome e idade de duas pessoas.
Ao final mostrar uma mensagem com os nomes e a idade média entre essas pessoas,
com uma casa decimal, conforme exemplo. */

#include<stdio.h>
#include<stdlib.h>

int main(){

    char nome1[50];
    char nome2[50];
    int idade1, idade2;
    double media;

    printf("Dados da primeira pessoa: \n");
    printf("Nome: ");
    scanf(" %[^\n]s",nome1);
    printf("Idade: ");
    scanf("%d",&idade1);

    printf("Dados da segunda pessoa: \n");
    printf("Nome: ");
    scanf(" %[^\n]s",nome2);
    printf("Idade: ");
    scanf("%d",&idade2);

    media = (double)(idade1 + idade2) / 2;

    printf("A idade média de %s e %s é de %.1lf anos",nome1, nome2, media);



    return 0;
}

File: Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaSequencial/src/application/ProblemaIdades.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaIdades {

	public static void main(String[] args) {
		/*
		 * Problema "idades"
Fazer um programa para ler o nome e idade de duas pessoas. Ao final mostrar uma mensagem com os
nomes e a idade média entre essas pessoas, com uma casa decimal, conforme exemplo.
		 */
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		String nome1, nome2;
		int idade1, idade2;
		double media;
		
		System.out.println("Dados da primeira pessoa:");
		System.out.print("Nome: ");
		nome1 = sc.nextLine();
		System.out.print("Idade: ");
		idade1 = sc.nextInt();
		System.out.println("Dados da segunda pessoa: ");
		sc.nextLine();
		System.out.print("Nome: ");
		nome2 = sc.nextLine();
		System.out.print("Idade: ");
		idade2 = sc.nextInt();
		media = (idade1 + idade2)/2.0;
		System.out.println("A idade média de "+nome1+" e "+nome2+" e de "+String.format("%.1f", media)+ " anos");
		
		
		sc.close();
	}

}
